backend/EasyMeets.Core/EasyMeets.Core.DAL/Entities/AdvancedSlotSettings.cs

```diff
@@ -0,0 +1,18 @@
+using EasyMeets.Core.DAL.Entities.Enums;
+
+namespace EasyMeets.Core.DAL.Entities;
+
+public class AdvancedSlotSettings : Entity<int>
+{
+    public int AvailabilitySlotId { get; set; }
+    public ActivityType ActivityType { get; set; }
+    public int Days { get; set; }
+    public DateTime StartDate { get; set; }
```

Comment:
It would be better to use DateTimeOffSet instead of DateTime (this is for all dates)